src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,5 +1,34 @@
 import breakpointsTheme from '@/components/UiBreakpoints';
 import colorTheme from '@/components/UiColorTheme';
+import { media } from '@/components/UiBreakpoints/Theme/media';
+
+const bp = breakpointsTheme.breakpoints.values;
```

ESLint

[error] Expected bp to have a type annotation.

(@typescript-eslint/typedef)